index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
```

Comment:
It's not super easy to do, but you do have several of these table ideas that are very similar. I'm not a fan of writing new javascript that spits out html. Doing it properly would probably be building up an angular component that takes the flavor text and knows how to structure the table, but I'm not exactly sure.